program-states/src/state/deposit_entry.rs

```diff
@@ -21,9 +21,10 @@ pub struct DepositEntry {
     pub voting_mint_config_idx: u8,
     // True if the deposit entry is being used.
     pub is_used: bool,
-    pub _reserved1: [u8; 6],
+    pub delegate_last_update_ts: u64,
+    pub _reserved1: [u8; 8],
```

Comment:
Why are we introducing the misalignment here? voting_mint_config_idx and is_used are still taking 1 byte each. Previously it was aligned to 8 bytes by using this array, now it's not. The size below also seems to be interesting to say the least. What this does it "packs" the voting_mint_config_idx and is_used into the same 8 byte range (leaving 6 bytes empty), then it puts your new u64 and then the new array of 8. So you end up with 14 extra empty bytes. To avoid this keep the u64s together and the smaller types near the "reserved" alignment.